src/server/db/seed/seedData.json

```diff
@@ -13,24 +13,209 @@
         {
             "email": "tombombadil@oldforest.me",
             "username": "OldTomBombadillo",
-            "password": "bright_blue_his_jacket_is_and_his_boots_are_yellow"
+            "password": "bright_blue_his_jacket_is_and_his_boots_are_yellow",
+            "isAdmin": true
+        },
+        {
+            "email": "sauron@mordor.me",
+            "username": "TheDarkLord",
+            "password": "one-ring-to-rule-them-all"
```

Comment:
Will isAdmin just default to false if it's not explicitly set? Or was this an oversight?

Reply:
Nvm looking at your screenshot it seems that is in fact the case

Reply:
Yeah, I set the default to false in the schema